master/security/http-methods.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Using HTTP methods and paths in policy rules
+---
+
+### Big picture
+
+Use Calico network policy for Istio-enabled apps to restrict ingress traffic that match HTTP methods or paths. 
+
+### Value
+
+Customer-facing applications like web apps can benefit from selective restrictions on ingress traffic (for example, HTTP GET requests). 
```

Comment:
Is there no value for east-west (inside the cluster) traffic as well? I feel like this value statement isn't complete.

Reply:
Better now?